src/test/java/no/nav/bilag/BilagITest.java

```diff
@@ -46,18 +47,18 @@ public class BilagITest {
 	private static final String AZURE_ENDEPUNKT = "/azure/token";
 	private static final String ACCESS_TOKEN_RESPONSE_BODY = """
 			{
-			  "access_token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6Ii1LSTNROW5OUjdiUm9meG1lWm9YcWJIWkdldyJ9.eyJhdWQiOiJiNTc5ZjM3OC1kMGZmLTRjZGUtOWVlYy0zZTNlODFlNzQ3NGUiLCJpc3MiOiJodHRwczovL2xvZ2luLm1pY3Jvc29mdG9ubGluZS5jb20vOTY2YWM1NzItZjViNy00YmJlLWFhODgtYzc2NDE5YzBmODUxL3YyLjAiLCJpYXQiOjE2NzMyNzMxMzUsIm5iZiI6MTY3MzI3MzEzNSwiZXhwIjoxNjczMjc4MTE3LCJhaW8iOiJBVFFBeS84VEFBQUF1NU5EaXpKMDBSSS9zZkpiYUJod1VOSm5uQktzY2JoRGN5K204TWNqY1ExYVFYZXJyN3FvMU5sUGxuajVBb1lpIiwiYXpwIjoiYjU3OWYzNzgtZDBmZi00Y2RlLTllZWMtM2UzZTgxZTc0NzRlIiwiYXpwYWNyIjoiMSIsImdyb3VwcyI6WyJkZWMzZWU1MC1iNjgzLTQ2NDQtOTUwNy01MjBlOGYwNTRhYzIiXSwibmFtZSI6IkZfWjk5NDA1OSBFX1o5OTQwNTkiLCJvaWQiOiI3ZDA4YzczMS1mNGYwLTRhNzQtYTNjOS03NWRkYTZlMjg5NWEiLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJGX1o5OTQwNTkuRV9aOTk0MDU5QHRyeWdkZWV0YXRlbi5ubyIsInJoIjoiMC5BVWNBY3NWcWxyZjF2a3VxaU1ka0djRDRVWGp6ZWJYXzBONU1udXctUG9IblIwNUhBUDAuIiwic2NwIjoiZGVmYXVsdGFjY2VzcyIsInN1YiI6InpZR0xFbndvVFNfdmtRclBxY0VsaEJ3SkpBU3d4eFB0cllNSjEtX1h6UTgiLCJ0aWQiOiI5NjZhYzU3Mi1mNWI3LTRiYmUtYWE4OC1jNzY0MTljMGY4NTEiLCJ1dGkiOiJNV1A1UW1pbklrYTZZNjFuSjFrM0FnIiwidmVyIjoiMi4wIiwiTkFWaWRlbnQiOiJaOTk0MDU5IiwiYXpwX25hbWUiOiJkZXYtZnNzOnRlYW1kb2t1bWVudGhhbmR0ZXJpbmc6ZG9rbWV0In0.NhFn9sHpdprRl_3GNBQplQEQIZ4RvWC4oYQdQ_7Q0vTey9tE7pZaNW3kGLnZYqO-LeegZJ1AAM1ddwivLOivhomL5lNyzM3nQORy4vKuZ9UXLpb3L-RXqyVs2KW4mPvhNQ1xPmNzFGEm1jOmuBFcJDkP8wbwXMXTJtS53oBBqOLK7jrcv6qnS0TATMHMdm6oHA4rXZcUlGfX__se1D9PY4g90QHkmpt6BcQyYdXkp7R5h21BVSM6VZ2AMA0f3DuudllvcgB_RyoJ9Bc1QUiArHiDVjFsIumWUCGryUKyTLS9NFBM0tFSTuJP7G8KGidQafLa5s8ZXD1sWaK_yWzsbQ",
+			  "access_token":"dummy-token",
```

Comment:
Push protection/secret scanning trigga på denne. Har endra til dummy verdi og markert som "Used in tests", men mulig den fortsatt gjør det.